concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -5321,6 +5321,53 @@ jobs:
           trigger: true
           passed: ['post-deploy']
 
+      # this tests the "integration" of this paas-billing revision with the
+      # configuration in this paas-cf revision, performed before deployment
+      # to give us a chance to avoid deploying a broken combination.
+      - task: paas-billing-integration-tests
+        tags: [colocated-with-web]
+        config:
+          platform: linux
+          image_resource: *cf-acceptance-tests-image-resource
+          params:
+            AWS_REGION: ((aws_region))
+            DEPLOY_ENV: ((deploy_env))
+          inputs:
+            - name: paas-cf
+            - name: paas-billing
+              path: src/github.com/alphagov/paas-billing
+          run:
+            path: sh
+            args:
+              - -e
+              - -u
+              - -c
+              - |
+                mkdir ./src/github.com/alphagov/paas-billing/gherkin/features/
+
+                cp "paas-cf/config/billing/output/${AWS_REGION}.json" ./src/github.com/alphagov/paas-billing/config.json
+                cp "paas-cf/config/billing/tests/${AWS_REGION}_billing_rds_charges.feature" ./src/github.com/alphagov/paas-billing/gherkin/features/
+
+                export APP_ROOT="$PWD"
+
+                wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+                echo "deb http://apt.postgresql.org/pub/repos/apt/ bionic-pgdg main" | tee /etc/apt/sources.list.d/pgdg.list
+
+                apt-get -q update
+                apt-get -q -y install postgresql-12 postgresql-client-12
```

Comment:
Do we really want to install postgres every pipeline run rather than including it in the docker image? I know we did that before, but now we are doing it twice.

Reply:
Not doing it twice - I've removed it from the latter task. But yeah, I'm trying (lol) to keep my changes focused. My guess is that they didn't go down the route of adding it to the docker image because it was seen as a temporary stopgap, and didn't want to load up our acceptance tests docker image with a postgres.